src/commands/build.ts

```diff
@@ -347,15 +347,25 @@ export async function command(commandOptions: CommandOptions) {
     const proxiedCode = await fs.readFile(proxiedFileLoc, {encoding: 'utf8'});
     const proxiedExt = path.extname(proxiedFileLoc);
     const proxiedUrl = proxiedFileLoc.substr(buildDirectoryLoc.length);
-    const proxyCode = await wrapCssModuleResponse(proxiedUrl, proxiedCode, proxiedExt);
+    const proxyCode = await wrapCssModuleResponse({
+      url: proxiedUrl,
+      code: proxiedCode,
+      ext: proxiedExt,
+      buildOptions: config.buildOptions,
```

Comment:
I have a slight preference for passing around the full `config` option whenever we need access to a full slice of config, instead of destructuring different parts of it. Especially in TypeScript, it's less mental overhead to only keep track of a single config type.

Would also be +1 with just passing the single `metaDir` string as an argument instead of any config objects.

Will leave it up to you how to resolve, if you have strong feels either way.

Reply:
Yeah passing the full config makes sense. I did think about the individual `metaDir` string, but I wanted to keep the mental mapping to where that comes from, as well as reduce noise if something else was needed from config. To that end the full config makes more sense!